packages/sdk-react/src/components/auth/Auth.tsx

```diff
@@ -223,12 +229,18 @@ const Auth: React.FC<AuthProps> = ({
 
       const sessionResponse = await passkeyClient?.createReadWriteSession({
         targetPublicKey: authIframeClient?.iframePublicKey!,
+        ...(authConfig.sessionLengthSeconds !== undefined && {
```

Comment:
tiny nit: `authConfig.sessionLengthSeconds &&...` would also be fine given `authConfig.sessionLengthSeconds` would be truthy if it's defined

also could make sense to update the README / have an example of this field being set https://github.com/tkhq/sdk/tree/main/packages/sdk-react#authentication

non-blocking though